exasol/bucketfs/_path.py

```diff
@@ -272,6 +287,9 @@ def parent(self) -> str:
     def as_uri(self) -> str:
         return self._path.as_uri()
```

Comment:
@Nicoretti @ahsimb This will be a file uri of the form file:///path/to/somewhere is this the intention

Reply:
Yes, it will return 'file:///...'. Whether we need it or not, I don't know.